src/otx/v2/adapters/torch/lightning/modules/datasets/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""OTX Algorithms - Dataset."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from .dataset import (
+    OTXVisualPromptingDataModule,
+    OTXVisualPromptingDataset,
+    get_transform,
+)
```

Comment:
This does not fit into a single line?
```suggestion
from .dataset import OTXVisualPromptingDataModule, OTXVisualPromptingDataset, get_transform
```

Reply:
Yes, I'll try that.